package-res/ccc/core/base/chart/chart.axes.js

```diff
@@ -433,6 +456,30 @@ pvc.BaseChart
             }
         }
 
+        // CDF603
```

Comment:
NOTE: the following "ifs" could probably be joined into more efficient nested "ifs" where, for example, min != null could be tested only once, at the outer level.